net/vpnc-scripts/files/vpnc-script

```diff
@@ -164,7 +164,11 @@ do_connect() {
 }
 
 do_disconnect() {
+	if [ -f "/tmp/dnsmasq.d/openconnect.$TUNDEV" ]; then
 	rm -f "/tmp/dnsmasq.d/openconnect.$TUNDEV"
```

Comment:
Can you please format the code correctly and indent the if?

Reply:
Yes, sorry my bad, miss that one